Lib/test/test_pydoc.py

```diff
@@ -913,7 +913,7 @@ def my_url_handler(url, content_type):
             text = 'the URL sent was: (%s, %s)' % (url, content_type)
             return text
 
-        serverthread = pydoc._start_server(my_url_handler, port=0)
+        serverthread = pydoc._start_server(my_url_handler, hostname='localhost', port=0)
         self.assertIn('localhost', serverthread.docserver.address)
```

Comment:
Technically this test does not validate that the patch is correct, since it would also pass without the code change!  But it’s probably fine for this patch.

Reply:
Is `localhost` specified anywhere else in the `pydoc.py` file? If it isn't then while you're right it isn't testing a difference from the default from the public API, this does help make sure that at least the value is being used.

Reply:
@brettcannon just making sure you don't want me to do anything here? The one place that 'localhost' is used in pydoc.py is for the default vaule of the hostname parameter for the `browse()` function to not break backwards compatibility.

Reply:
See my comment before Brett’s: technically, this test does not prove that the patch is correct, since `'localhost'` is the value that was used before your changes.